lib/checkRoutes.js

```diff
@@ -0,0 +1,41 @@
+require('colors')
+const fs = require('fs-extra')
+const path = require('path')
+const logger = require('./tools/logger')()
+
+// Utility module for checking express routes in controllersPath
+module.exports = function (app, callback) {
+  const controllersPath = app.get('controllersPath')
+  let badControllers = []
+  let controllerFiles
+
+  // Make sure controllersPath is a directory
+  try {
+    fs.accessSync(controllersPath)
+
+    // Get all controller files
+    controllerFiles = fs.readdirSync(controllersPath)
+
+    // Loop through files and their content
+    for (let controller of controllerFiles) {
+      let file = path.join(controllersPath, controller)
+      let content = fs.readFileSync(file, 'utf8')
+
+      // If no express routes are found (error files may not have an app.route())
+      if (!content.match(/\.(route|status|send)*?\(([^)]*)\)/)) {
```

Comment:
i think there might be some false negatives here

take a look at the [various ways to write express routes](https://expressjs.com/en/guide/routing.html)

wouldn't this test fail something like this?

```javascript
  app.get('/stuff', (req, res) => {
    res.render('something', {})
  })
```